src/v/raft/group_configuration.cc

```diff
@@ -110,6 +110,32 @@ bool group_configuration::is_voter(vnode id) const {
     return old_it != std::cend(_old->voters);
 }
 
+bool group_configuration::is_allowed_to_request_votes(vnode id) const {
+    // either current voter
+    auto it = std::find(
+      std::cbegin(_current.voters), std::cend(_current.voters), id);
```

Comment:
nit: using `std::begin` I think only really makes sense if you're writing code that is templated on the container type.

Reply:
i will migrate this in separate PR.

Reply:
Follow up: https://github.com/vectorizedio/redpanda/pull/3063